app/views/topics/_list.html.erb

```diff
@@ -0,0 +1,30 @@
+<tbody>
+  <% @topics.each do |topic| %>
+    <tr>
+      <td class="text-bold-500"><%= topic.title %></td>
+      <td class="text-bold-500"><%= topic.description.truncate(25, omission: "...") %></td>
```

Comment:
Truncate needs a safe navigation operator. It will throw an undefined method for nil error if description is not provided.

Reply:
Our existing tests will catch this if we set only the required fields in the topics factory.

Reply:
We also have an errant not-null on description and UID in the schema. Both of those should be optional.

Reply:
So should we make description and title nullable?

Reply:
I got that in the import branch, so we're good.

Reply:
Actually, since this is pulled into another file, we'll need to get it here. I got the migration portion but we'll need to add the save navigation operator before those two truncates. Then I think we're good to go.